src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -61,613 +70,58 @@ private Http2LoopbackServer(Http2Options options)
-
-            // First read the frame headers, which should tell us how long the rest of the frame is.
-            byte[] headerBytes = new byte[Frame.FrameHeaderLength];
-
-            try
-            {
-                if (!await FillBufferAsync(headerBytes, timeoutCts.Token).ConfigureAwait(false))
-                {
-                    return null;
-                }
-            }
-            catch (IOException)
-            {
-                // eat errors when client aborts connection and return null.
-                return null;
-            }
-
-            Frame header = Frame.ReadFrom(headerBytes);
-
-            // Read the data segment of the frame, if it is present.
-            byte[] data = new byte[header.Length];
-            if (header.Length > 0 && !await FillBufferAsync(data, timeoutCts.Token).ConfigureAwait(false))
-            {
-                throw new Exception("Connection stream closed while attempting to read frame body.");
-            }
-
-            if (_ignoreSettingsAck && header.Type == FrameType.Settings && header.Flags == FrameFlags.Ack)
-            {
-                _ignoreSettingsAck = false;
-                return await ReadFrameAsync(timeout).ConfigureAwait(false);
-            }
-
-            if (_ignoreWindowUpdates && header.Type == FrameType.WindowUpdate)
-            {
-                return await ReadFrameAsync(timeout).ConfigureAwait(false);
-            }
-
-            // Construct the correct frame type and return it.
-            switch (header.Type)
-            {
-                case FrameType.Data:
-                    return DataFrame.ReadFrom(header, data);
-                case FrameType.Headers:
-                    return HeadersFrame.ReadFrom(header, data);
-                case FrameType.Priority:
-                    return PriorityFrame.ReadFrom(header, data);
-                case FrameType.RstStream:
-                    return RstStreamFrame.ReadFrom(header, data);
-                case FrameType.Ping:
-                    return PingFrame.ReadFrom(header, data);
-                case FrameType.GoAway:
-                    return GoAwayFrame.ReadFrom(header, data);
-                default:
-                    return header;
-            }
-        }
-
-        // Reset and return underlying networking objects.
-        public (Socket, Stream) ResetNetwork()
+        private void RemoveInvalidConnections()
         {
-            Socket oldSocket = _connectionSocket;
-            Stream oldStream = _connectionStream;
-            _connectionSocket = null;
-            _connectionStream = null;
-            _ignoreSettingsAck = false;
-
-            return (oldSocket, oldStream);
+            _connections.RemoveAll((c) => c.IsInvalid);
         }
 
         // Returns the first 24 bytes read, which should be the connection preface.
-        public async Task<string> AcceptConnectionAsync()
+        public async Task<Http2LoopbackConnection> AcceptConnectionAsync()
```

Comment:
Update comment above

Reply:
removed it since it's not useful after updating